### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by luis-pomare**
In this milestone, we have:

- Created the database (vet_clinic)
- Created one table (animals)
- Made the insertion of some animals in the "animals" table
- Executed the following queries:
    - Find all animals whose name ends in "mon":
    
image

    - List the name of all animals born between 2016 and 2019:
    
image

    - List the name of all animals that are neutered and have less than 3 escape attempts:
    
image

    - List the date of birth of all animals named either "Agumon" or "Pikachu":
    
image

    - List name and escape attempts of animals that weigh more than 10.5kg:
    
image

    - Find all animals that are neutered:
    
image

    - Find all animals not named Gabumon:
    
image

    - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg):
    
image

### CODE REVIEW

**Comment by omarsalem7** [comment]
Hi @luis-pomare 😊, 

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 💀 

You have submitted a project, that hasn't correct git-flow, you should open a PR from the feature branch to the development branch, not the main. Also, consider customizing the readme for this project and removing the placeholder text and making sure to include all sections in the readme and fill them with related info

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by rotimiazeez — COMMENTED** [review]
# Hi @luis-pomare, 

### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that did not have a professional README file. Please fill out all the sections available in the README file before submitting for review 🙏 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,15 @@
-/* Database schema to keep the structure of entire database. */
+-- Database schema to keep the structure of entire database.
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE DATABASE vet_clinic;
+CREATE TABLE animals(
+    id INT,
```
Comment (Nyame-Wolf) [inline schema.sql:5]: [OPTIONAL] I suggest you make this column your primary key. Having primary keys will assist you link to other table by using your primary keys as foreign key of other tables.

**Review by Nyame-Wolf — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi <STUDENT_MENTION> :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :sparkles:

- [ ] Professional readme 👍🏾 
- [ ] used correct project structure

## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**